packages/react-components/react-card/src/components/Card/Card.types.ts

```diff
@@ -1,13 +1,19 @@
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
 
+/**
+ * Card refs to the root element slot.
+ */
+
+export type CardRefElement = HTMLDivElement | HTMLButtonElement | HTMLAnchorElement;
+
 /**
  * Slots available in the Card component.
  */
 export type CardSlots = {
   /**
    * Root element of the component.
    */
-  root: Slot<'div'>;
+  root: Slot<'div', 'a' | 'button'>;
```

Comment:
nit: would be nice to have a little comment of those options as an enhancement for interaction.

Reply:
Good one. Updated that!